src/en/installconfig-network.md

```diff
@@ -0,0 +1,41 @@
+Title: Network Configuration
```

Comment:
I think perhaps the best thing to do is for me to raise a couple of issues for these rewrites (to push the GUI) so we can get the new repository populated with the old docs as quickly as possible, before then going back and tackling the issues themselves.